kivy/uix/behaviors/focus.py

```diff
@@ -489,6 +487,22 @@ def _get_focus_next(self, focus_dir):
             else:
                 return None
 
+    def get_focus_next(self):
+        '''This method provides the next focuses(ie self.focus_next)
```

Comment:
@mantham sure, i didn't pick up on that , my mistake.I will surely act on them and get them rectified.